app/assets/stylesheets/custom.scss

```diff
@@ -155,6 +157,52 @@ th {
   text-align: center;
 }
 
+// about page
+
+.about {
+  background-color: #e3e3e3;
+  border: 2px solid #6699ff;
+  border-radius: 20px;
+  margin-bottom: 2px;
+  min-height: 20px;
+  padding: 50px;
+}
+
+// footer
+footer {
+  background-color: #bcdefa;
+  border-top: 1px solid #eaeaea;
+  bottom: 0;
+  height: 60px;
+  position: relative;
+  width: 100%;
+}
+
+footer a {
+  color: #674388;
+  margin-right: 15px;
+  font-size: 1.3em;
+}
+
+footer a:hover {
+  color: #222;
+}
+
+footer small {
+  float: left;
+  font-size: 0.9em;
+}
+
+footer ul {
+  float: right;
+  text-decoration: underline;
+}
+
+footer ul li {
```

Comment:
This can be compressed. Hartl goes in how to refactor CSS. It is complaining that you are breaking out CSS declarations on multiple lines when they can be put into one line.